src/routes/docs/tutorials/nuxt/step-5/+page.markdoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Add navigation
+description: Add navigation to your app.
+step: 5
+---
+
+To help our users navigate the app we want it to have a navigation bar that's visible on all pages.
+We will once again use the `useUserSession()` composable for information about the current user.
+
+With this piece of information we will show a login button when no user is logged in and a logout button when one is.
+We will also put the user's e-mail address next to the logout button.
+
+From the `components` directory, create the file `navbar.vue` and add the code below.
+
+```vue
+// components/navbar.vue
+
+<template>
+  <div>
+ <!--- Navbar -->
+    <nav class="main-header u-padding-inline-end-0">
+      <h3 class="u-stretch eyebrow-heading-1">Idea Tracker</h3>
+      <!-- Email and logout button if logged in user -->
+      <div
+        class="main-header-end u-margin-inline-end-16"
+        v-if="user.isLoggedIn.value === true"
```

Comment:
IIRC you don't need .value in templates. Weird, I know.

Reply:
Fixed